drivers/remote/driver.go

```diff
@@ -146,6 +147,9 @@ func (d *driver) EndpointOperInfo(nid, eid types.UUID) (map[string]interface{},
 	if err := d.call("EndpointOperInfo", info, &res); err != nil {
 		return nil, err
 	}
+	if val, ok := res.Value["MacAddress"]; ok {
+		res.Value[netlabel.MacAddress], _ = net.ParseMAC(val.(string))
+	}
```

Comment:
@mbanikazemi agreed. any API/Structure between libnetwork core and drivers must be well-defined (not necessarily `netlabel` but atleast well-defined constants), to avoid such confusions. I understand that this will expand the scope of this PR. But will you mind taking care of that ?